examples/model_selection/plot_learning_curve.py

```diff
@@ -126,3 +126,128 @@ def plot_learning_curve(estimator, title, X, y, ylim=None, cv=None,
+from sklearn.datasets import make_regression
+from sklearn.linear_model import SGDRegressor
+from sklearn.svm import SVR
+from sklearn.neighbors import KNeighborsRegressor
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.model_selection import ShuffleSplit
+
+
+def plot_learning_curve_times(estimators, X, y, train_sizes,
+                              cv=None, n_jobs=None):
+    """
+    Generate a simple plot of the test and training learning curve.
+
+    Parameters
+    ----------
+    estimators : A list of object type that implement the "fit" and "predict"
+        methods. An object of that type which is cloned for each validation.
+
+    X : array-like, shape (n_samples, n_features)
+        Training vector, where n_samples is the number of samples and
+        n_features is the number of features.
+
+    y : array-like, shape (n_samples) or (n_samples, n_features), optional
+        Target relative to X for classification or regression;
+        None for unsupervised learning.
+
+    train_sizes : array-like, shape (n_ticks,), dtype float or int
+        Relative or absolute numbers of training examples that will be used to
+        generate the learning curve. If the dtype is float, it is regarded as a
+        fraction of the maximum size of the training set (that is determined
+        by the selected validation method), i.e. it has to be within (0, 1].
+        Otherwise it is interpreted as absolute sizes of the training sets.
+        Note that for classification the number of samples usually have to
+        be big enough to contain at least one sample from each class.
+
+    cv : int, cross-validation generator or an iterable, optional
+        Determines the cross-validation splitting strategy.
+        Possible inputs for cv are:
+          - None, to use the default 3-fold cross-validation,
+          - integer, to specify the number of folds.
+          - :term:`CV splitter`,
+          - An iterable yielding (train, test) splits as arrays of indices.
+
+        For integer/None inputs, if ``y`` is binary or multiclass,
+        :class:`StratifiedKFold` used. If the estimator is not a classifier
+        or if ``y`` is neither binary nor multiclass, :class:`KFold` is used.
+
+        Refer :ref:`User Guide <cross_validation>` for the various
+        cross-validators that can be used here.
+
+    n_jobs : int or None, optional (default=-1)
+        Number of jobs to run in parallel.
+        ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
+        ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
+        for more details.
+    """
+    fig, axes = plt.subplots(1, 2, figsize=(18, 6))
+
+    axes[0].set_title("Learning Curves computation times")
+    axes[0].set_xlabel("Training examples")
+    axes[0].set_ylabel("Training time (s)")
+    axes[0].grid()
+
+    axes[1].set_title("Estimators fit times with scores")
+    axes[1].set_xlabel("Fit times (s)")
```

Comment:
What's the difference between "fit time" and "training time"??

Reply:
It depends on the definition of training time:
- we can say `training time = fit time` or
- we can say `training time = fit time + score time`
Note that with these changes, we can also plot training examples against `score times`. 
In the example I chose to plot only training examples against `fit times`